### PR TITLE
Fix(UI)#6445: Global Search no data placeholder removal

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on the global search to remove the 'No Data Found' message and logo showing initially.
I also added a loader to show in suggestions while searching and removed the empty space below search bar.

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">

https://user-images.githubusercontent.com/51777795/182196843-1c6bfcee-f035-44e7-81ed-db7778e6caf1.mov

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/GlobalSearchProvider/GlobalSearchSuggestions/GlobalSearchSuggestions.tsx [hunk 1]
```diff
@@ -307,10 +313,17 @@ const GlobalSearchSuggestions = ({
         listHeight={220}
         notFoundContent={
           <div className="tw-flex tw-flex-col tw-w-full tw-h-56 tw-items-center tw-justify-center tw-pb-9">
-            <NoDataFoundSVG />
-            <Typography>No Results Found</Typography>
+            {isSuggestionsLoading ? (
+              <Loader size="small" />
+            ) : (
+              <>
+                <NoDataFoundSVG />
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/GlobalSearchProvider/GlobalSearchSuggestions/GlobalSearchSuggestions.tsx:320]: Can we use antd `Empty` component here?
https://ant.design/components/empty/#header

File: openmetadata-ui/src/main/resources/ui/src/components/GlobalSearchProvider/GlobalSearchProvider.tsx [hunk 2]
```diff
@@ -124,6 +132,7 @@ const GlobalSearchProvider: FC<Props> = ({ children }: Props) => {
           onSearch={(newValue) => {
             debounceOnSearch(newValue);
             setSearchValue(newValue);
+            setIsSuggestionsLoading(true);
```
[the remark this answers is not in the document]
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/GlobalSearchProvider/GlobalSearchProvider.tsx:135]: Yes, debouncing that results is delay in showing the loader and 'No Data' will be displayed during that.

**Comment by vivekratnavel** [comment]
@aniketkatkar97 Should we reserve that empty space below the search bar when "Cmd + k" is pressed?

**Comment by aniketkatkar97** [comment]
> @aniketkatkar97 Should we reserve that empty space below the search bar when "Cmd + k" is pressed?

Sure, will look into it.

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by vivekratnavel — APPROVED** [review]

**Pull request merged**